src/player.cpp

```diff
@@ -5502,7 +5502,8 @@ void player::suffer()
         const mutation_branch &mdata = m.first.obj();
         for( const body_part bp : all_body_parts ) {
             if( calendar::once_every( 1_minutes ) ) {
-                const float wetness_percentage =  body_wetness[bp] / drench_capacity[bp]; // 0.0 - 1.0
+                const float wetness_percentage =  static_cast<float>( body_wetness[bp] ) / static_cast<float>
```

Comment:
You only need to explicitly cast one of the operands. The other one will be casted automatically.